### PR TITLE
fix(1500): Make queue-service available for SD_ZIP_ARTIFACTS

### PR DESCRIPTION
**Pull request by kkisic**
## Context
Implement `/v1/queue/message?type=unzip` endpoint for SD_ZIP_ARTIFACTS in order to make queue-service available to that function.

## Objective
- Create unzip queue for SD_ZIP_ARTIFACTS job
- Implements `scheduler.unzipArtifacts()`

## References
https://github.com/screwdriver-cd/screwdriver/issues/1500
https://github.com/screwdriver-cd/screwdriver/pull/2583

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: plugins/queue/scheduler.js [hunk 1]
```diff
@@ -688,6 +689,32 @@ async function clearCache(executor, config) {
     }
 }
 
+/**
+ * Pushes a message to unzip artifacts
+ * @async unzipArtifacts
+ * @param {Object} executor
+ * @param {Object} config               Configuration
+ * @param {String} config.buildId       Unique ID for a build
+ */
```
Comment (sugarnaoming) [inline plugins/queue/scheduler.js:698]: It seems that you forgot to write the JSDoc of the return value.

Comment (kkisic) [reply plugins/queue/scheduler.js:699]: Thx.  I fixed it.

**Review by yuichi10 — APPROVED** [review]
LGTM

**Review by sugarnaoming — APPROVED** [review]
LGTM

**Review by tkyi — APPROVED** [review]

**Pull request merged**